README.md

```diff
@@ -295,22 +306,15 @@ The Client ID and the Client Secret will be needed for the basic script you will
 
 ### Flexy OCS Configuration File Changes
 1. Add OCS information to the ConnectorConfig.json file.
-   1. Replace the xxx below with the namespace.
-   1. Replace the yyy below with the tenant ID.
-   1. Copy these into the ServerConfig portion of the json file.
-      ```json
-      "Namespace":"xxx",
-      "TenantId":"yyy"
-      ```
+   1. Replace the {tenantId} below with the tenant ID of the OCS account.
+   1. Replace the{namespace} below with the namespace of the OCS account.
```

Comment:
not sure if a space is needed  after the.

Reply:
added